externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/SparkSQLEngineListenerSuite.scala

```diff
@@ -20,7 +20,7 @@ package org.apache.kyuubi.engine.spark
 import org.apache.kyuubi.service.ServiceState
 
 class SparkSQLEngineListenerSuite extends WithSparkSQLEngine {
-  override def conf: Map[String, String] = Map.empty
+  override def withKyuubiConf: Map[String, String] = Map.empty
```

Comment:
I believe this name is more readable.

Reply:
It's not only Kyuubi confs, right?

Reply:
The config mush with the prefix `kyuubi.` so that kyuubi could load.

Reply:
kyuubiConf.set is different, it accepts all pattern.
FYI, the iceberg configs are not `kyuubi.`-prefixed
https://github.com/yaooqinn/kyuubi/pull/387/files#diff-ebc4d984b839029afebf66e9c0fed77837f88b9a32fc81831dc681d6fc001dfaR25

Reply:
ah, yes we can load that. `withKyuubiConf` mean the conf will be loaded into kyuubi conf.